## **Equipo 7**

**Integrantes:**

- Camacho Herrera Jesús Salvador

- Flores Solis Eduardo Elías

- Garcia Robles Viviana

- Mendoza López Luis Ángel

## **Dependencias**

In [1]:
import numpy as np
import scipy.stats as stats

## **Problemas**

### **Problema 2**

Una muestra aleatoria $X_{1}, \ldots, X_{100}$ se da a partir de una distribución con varianza conocida $\mbox{Var}(X_{i})=16$. Para la muestra observada, la media muestral es $\overline{X}=23.5$. Encuentre un intervalo de confianza aproximado del 95% para $\theta = \mathbb{E}[X_{i}]$.

#### **Solución**

Por lo visto en clase, aquí el intervalo de confianza del $(1−α)100\%$ es

$$IC_{\alpha} = \left[\overline{X}_{n} - z_{\frac{\alpha}{2}}\frac{\sigma}{\sqrt{n}}, \overline{X}_{n} + z_{\frac{\alpha}{2}}\frac{\sigma}{\sqrt{n}}\right]$$

Veamos el cálculo con python:

In [2]:
# Calculando z_alpha
alpha = 1 - 0.95
z_alpha = stats.norm.ppf(1-alpha/2)
z_alpha

1.959963984540054

Así que

$$ z_{\frac{\alpha}{2}} = \Phi^{-1}_{N(0,1)}\left(\frac{\alpha}{2}\right) ≈ 1.96$$

In [3]:
# Obteniendo los extremos del intervalo
IC = (23.5 - z_alpha * 4/np.sqrt(100), 23.5 + z_alpha * 4/np.sqrt(100))
IC

(22.71601440618398, 24.28398559381602)

Por lo tanto, el intervalo de confianza del $95\%$ es

$$IC_{\alpha} ≈ [22.71, 24.28]$$

### **Problema 3**

Para estimar la proporción de votantes que planean votar por el Candidato A en una elección, se elige una muestra aleatoria de tamaño $n$ de los votantes. El muestreo se realiza con reemplazo. Sea $\theta$ la proporción de votantes que planean votar por el Candidato A entre todos los votantes. ¿Qué tan grande debe ser $n$ para que podamos obtener un intervalo de confianza del $90%$ con un margen de error del $3\%$? Es decir, ¿qué tan grande debe ser $n$ para que

$$\mathbb{P}\left(\overline{X}-0.03 \leq \theta \leq \overline{X}+0.03\right) \geq 0.90,$$

donde $\overline{X}$ es la proporción de personas en nuestra muestra aleatoria que dicen que planean votar por el Candidato A.

#### **Solución**

En este caso el intervalo del $(1-\alpha)100\%$ de confianza está dado como

$$IC_{\alpha} = \left[\overline{X}_{n} - \frac{z_{\frac{\alpha}{2}}}{2\sqrt{n}}, \overline{X}_{n} + \frac{z_{\frac{\alpha}{2}}}{2\sqrt{n}}\right]$$

Veamos el cálculo con python:

In [4]:
alpha = 1 - 0.90
z_alpha = stats.norm.ppf(1-alpha/2)
z_alpha

1.6448536269514722

In [5]:
# Y queremos un margen de error del 3%
error = 0.03
n = (z_alpha/(error*2))**2
n

751.5398483598369

Así que $n \geq 752$ es suficiente.

### **Problema 4**

**a)** Sea $X$ una variable aleatoria tal que $R_{X} \subset [a, b]$, es decir, tenemos que $a \leq X \leq b$. Mostrar que

$$ \mbox{Var}(X) \leq \frac{(b-a)^{2}}{4} $$

**b)** Sea $X_{1}, \ldots, X_{n}$ una muestra aleatoria de una distribución desconocida con función de distribución acumulada $F_{X}(x)$ tal que $R_{X} \subset [a, b]$. Específicamente, $\mathbb{E}(X)$ y $\mbox{Var}(X)$ son desconocidas. Encuentre un intervalo de confianza de $(1-\alpha)100\%$ para $\theta = \mathbb{E}(X)$. Supongamos que $n$ es grande.

#### **Solución**

**a)** Definamos a la siguiente v.a.
$$ Y = X - \frac{a+b}{2} $$

Por tanto

$$-\frac{b-a}{2}=a -  \frac{a+b}{2} \leq Y \leq b - \frac{a+b}{2}=\frac{b-a}{2}$$

Es decir, $R_{Y} ⊂ \left[-\frac{b-a}{2}, \frac{b-a}{2}\right]$.

Así tenemos que

\begin{align}
\mbox{Var}(X) &= \mbox{Var}(Y) \\
 &= \mathbb{E}(Y^{2}) -  (\mathbb{E}(Y))^{2}\\
 &\leq \mathbb{E}(Y^{2}) \\
 &\leq \left(\frac{b-a}{2}\right)^{2} \\
 &= \frac{(b-a)^{2}}{4}
\end{align}

La primer igualdad se sigue por propiedades de la varianza, y la segunda desigualdad es por propiedades de la esperanza. $\quad \blacksquare$

**b)** Por el inciso anterior tenemos una cota superior para $\sigma$, que es $\sigma_{max} = \frac{b-a}{2}$ (pues para $\mbox{Var}(X)=\sigma^{2}$ es $\frac{(b-a)^{2}}{4}$).

Por lo tanto, el intervalo del $(1-\alpha)100\%$ de confianza está dado como


$$IC_{\alpha} = \left[\overline{X}_{n} - z_{\frac{\alpha}{2}}\frac{\sigma_{max}}{\sqrt{n}}, \overline{X}_{n} + z_{\frac{\alpha}{2}}\frac{\sigma_{max}}{\sqrt{n}}\right] = \left[\overline{X}_{n} - z_{\frac{\alpha}{2}}\frac{b-a}{2\sqrt{n}}, \overline{X}_{n} + z_{\frac{\alpha}{2}}\frac{b-a}{2\sqrt{n}}\right].$$

### **Problema 5**

Se toma una muestra aleatoria $X_{1}, \ldots, X_{144}$ de una distribución con varianza desconocida $\mbox{Var}(X)=\sigma^{2}$. Para la muestra observada, la media muestral es $\overline{X} = 55.2$, y la varianza de la muestra es $S^{2} = 34.5$. Encontrar un intervalo de confianza del $99\%$ para $\theta = \mathbb{E}(X_{i})$.

#### **Solución**

Como la varianza es desconocida en este caso el intervalo es

$$IC_{\alpha} = \left[\overline{X}_{n} - z_{\frac{\alpha}{2}}\frac{S}{\sqrt{n}}, \overline{X}_{n} + z_{\frac{\alpha}{2}}\frac{S}{\sqrt{n}}\right]$$

Hagamos los cálculos con python:

In [6]:
# Calcumos z
alpha = 1 - 0.99
z_alpha = stats.norm.ppf(1-alpha/2)
z_alpha

2.5758293035489004

In [7]:
# Obtenemos los límites del intervalo
IC = (55.2 - z_alpha * np.sqrt(34.5)/np.sqrt(144), 55.2 + z_alpha * np.sqrt(34.5)/np.sqrt(144))
IC

(53.939202377859694, 56.46079762214031)

Por lo tanto, el intervalo de confianza del $99\%$ es

$$IC_{\alpha} ≈ [53.93, 56.46]$$

### **Problema 6**

Una muestra aleatoria $X_{1}, \ldots, X_{16}$ de una distribución normal con media desconocida $\mu = \mathbb{E}(X_{i})$, y varianza desconocida $\mbox{Var}(X_{i})=\sigma^{2}$. Para la muestra observada, la media muestral es $\overline{X} = 16.7$, y la varianza de la muestra es $S^{2} = 7.5$.

a) Encontrar un intervalo de confianza del $95\%$ para $\mu$.

b) Encontrar un intervalo de confianza del $95\%$ para $\sigma^{2}$.

#### **Solución**

**a)**

Como la media y varianza son desconocidas, el intervalo para $\mu$ está dado de la siguiente forma

$$IC_{\alpha} = \left[\overline{X}_{n} - t_{\frac{\alpha}{2}, n-1}\frac{S}{\sqrt{n}}, \overline{X}_{n} + t_{\frac{\alpha}{2}, n-1}\frac{S}{\sqrt{n}}\right]$$

Veamos los cálculos:

In [8]:
# Calculando t
alpha = 1 - 0.95
t_alpha = stats.t.ppf(1-alpha/2, 15)
t_alpha

2.131449545559323

In [9]:
# Obteniendo el intervalo de confianza
IC = (16.7 - t_alpha * np.sqrt(7.5)/np.sqrt(16), 16.7 + t_alpha * np.sqrt(7.5)/np.sqrt(16))
IC

(15.240696254641279, 18.15930374535872)

Por lo tanto, el intervalo de confianza del $95\%$ para $\mu$ es

$$IC_{\alpha} ≈ [15.24, 18.15]$$

**b)**

Como la media y varianza son desconocidas, el intervalo para $\mu$ está dado de la siguiente forma

$$IC_{\alpha} = \left[\frac{(n-1)S^{2}}{χ^{2}_{\frac{\alpha}{2},n-1}}, \frac{(n-1)S^{2}}{χ^{2}_{1-\frac{\alpha}{2},n-1}} \right]$$

Veamos los cálculos:

In [10]:
# Calculemos chi_alpha
alpha = 1 - 0.95
chi_alpha_1 = stats.chi2.ppf(1-alpha/2, 15)
chi_alpha_2 = stats.chi2.ppf(alpha/2, 15)
chi_alpha_1, chi_alpha_2

(27.488392863442975, 6.262137795043253)

In [11]:
# Por último obtenemos el intervalo de confianza
IC = (15*7.5/chi_alpha_1, 15*7.5/chi_alpha_2)
IC

(4.092636501481853, 17.965110906541934)

Por lo tanto, el intervalo de confianza del $95\%$ para $\sigma^{2}$ es

$$IC_{\alpha} ≈ [4.09, 17.96]$$